### DISCUSSION TITLE
Making ot available as a shared library

### DISCUSSION THREAD

**Discussion by perry2Erip** [rhasspy/piper#433] [Ideas]
For my personal use cases I don't want to run a binary but I also don't want to use python. I'm sure I'm not the only one who wants it (see #276). A proper .so/.dll/.dylib (maybe even a static object version) implementation is something that can't be bad and will only help the use of Piper across a range of projects.

**Comment by danemadsen** [comment]
Im also trying to do this but i cannot get it to compile, as anyone else had any success?
